packages/core/entity/Annotation/index.ts

```diff
@@ -14,11 +17,21 @@ export interface AnnotationResponse {
     annotationDisplayName: string;
     annotationName: string;
     description: string;
-    type: string;
+    type: string | AnnotationType;
```

Comment:
hmm why potentially `string`? simplifying the type whenever possible seems ideal

Reply:
I think this is actually because DatabaseAnnotationService uses other internally defined types, e.g., `DatabaseService.OPEN_FILE_LINK_TYPE,`

Reply:
changed in [ceede0e](https://github.com/AllenInstitute/biofile-finder/pull/406/commits/ceede0e5f22524b7a9e21df69a046a1c1a810014)